src/collective/upgrade/run.py

```diff
@@ -116,10 +116,10 @@ def main(app=None, args=None):
     runner = app.restrictedTraverse('@@collective.upgrade.form')
     try:
         runner.upgrade(**kw)
-    except:
+    except Exception:
```

Comment:
Good additional catch here, thanks!

Reply:
No Problem 😅  Will there be a release for this or are there some other things coming before it will be released?

Reply:
You wanna do that?  I checked and [the `collective` PyPI user](https://pypi.org/user/collective/) has the Owner role for this project.  I forget how it works for those of us in the community to use that user account to upload releases, but I dimly recall there being some way to do that.  Maybe ask on what ever passes for the Plone community forums or chat these days?